semantic-conventions/syntax.md

```diff
@@ -0,0 +1,235 @@
+# Semantic Convention YAML Language
```

Comment:
Please add a note that when this file is changed one should also adapt `semconv.schema.json` accordingly.

Reply:
Done in f46daff, also added a TOC.